builder/Makefile.release

```diff
@@ -0,0 +1,109 @@
+MAKE_BUILDER_CONTAINER = $(CMD_MAKE) -f builder/Makefile.tracee-make ubuntu-make ARG="all"
+
+.check_%:
+#
+	@command -v $* >/dev/null
+	if [ $$? -ne 0 ]; then
+		echo "missing required tool $*"
+		exit 1
+	else
+		touch $@ # avoid target rebuilds due to inexistent file
+	fi
+
+.PHONY: .check_release_tag
+.ONESHELL:
+.check_release_tag:
+#
+	@if [ -z "$(RELEASE_TAG)" ]; then
+		echo "missing required variable RELEASE_TAG"
+		exit 1
+	fi
+
+#
+# release
+#
+release_notes := $(OUT_DIR)/release-notes.txt
+.PHONY: release
+release: \
+	| .check_release_tag \
+	.check_$(CMD_GIT) \
+	.check_$(CMD_DOCKER) \
+	.check_$(CMD_CHECKSUM) \
+	.check_$(CMD_GITHUB) \
+	.check_$(CMD_TAR) \
+	.check_$(CMD_CP) \
+#
+	-rm $(release_notes)
+	echo '## Changelog' > $(release_notes)
+	$(CMD_GIT) log --pretty=oneline --abbrev=commit --no-decorate --no-color tags/$(shell $(CMD_GIT) describe --tags --abbrev=0)..HEAD >> $(release_notes)
+	echo '' >> $(release_notes)
+#
+#   Create build artifact archive
+#
+	$(MAKE_BUILDER_CONTAINER_PREPARE)
+	BTFHUB=1 $(MAKE_BUILDER_CONTAINER)
+	$(CMD_TAR) -czf $(OUT_ARCHIVE) $(RELEASE_FILES)
+	$(CMD_CHECKSUM) $(OUT_ARCHIVE) > $(OUT_CHECKSUMS)
+#
+#   Create and push git tag
+#
+	$(CMD_GIT) tag -f $(RELEASE_TAG)
+	$(CMD_GIT) push -u origin $(RELEASE_TAG)
+#
+#	Build container images
+#
+	$(MAKE_CONTAINER) $(TARGET_BTFHUB_IMAGE_NAME)
+	$(MAKE_CONTAINER) $(TARGET_CORE_IMAGE_NAME)
+	$(MAKE_CONTAINER) $(TARGET_NOCORE_IMAGE_NAME)
+#
+#   Push archive and checksums to github (auth in github action workflow)
+#
+	$(CMD_GITHUB) release create $(RELEASE_TAG) $(OUT_ARCHIVE) $(OUT_CHECKSUMS) --title $(RELEASE_TAG) --notes-file $(release_notes)
+#
+#   Push images to dockerhub
+#
+	$(CMD_DOCKER) tag $(TARGET_BTFHUB_IMAGE_NAME) $(PUSH_DOCKER_REPO):latest
+	$(CMD_DOCKER) push $(PUSH_DOCKER_REPO):latest
+#	
+	$(CMD_DOCKER) tag $(TARGET_BTFHUB_IMAGE_NAME) $(PUSH_DOCKER_REPO):$(TARGET_BTFHUB_IMAGE_NAME)-$(RELEASE_TAG)
+	$(CMD_DOCKER) push $(PUSH_DOCKER_REPO):$(TARGET_BTFHUB_IMAGE_NAME)-$(RELEASE_TAG)
+#
+	$(CMD_DOCKER) tag $(TARGET_CORE_IMAGE_NAME) $(PUSH_DOCKER_REPO):$(TARGET_CORE_IMAGE_NAME)-$(RELEASE_TAG)
+	$(CMD_DOCKER) push $(PUSH_DOCKER_REPO):$(TARGET_CORE_IMAGE_NAME)-$(RELEASE_TAG)
+#
+	$(CMD_DOCKER) tag $(TARGET_NOCORE_IMAGE_NAME) $(PUSH_DOCKER_REPO):$(TARGET_NOCORE_IMAGE_NAME)-$(RELEASE_TAG)
+	$(CMD_DOCKER) push $(PUSH_DOCKER_REPO):$(TARGET_NOCORE_IMAGE_NAME)-$(RELEASE_TAG)
+#
+	echo '## Docker images' >> $(release_notes)
+	echo '- `docker pull docker.io/$(PUSH_DOCKER_REPO):latest (BTFHub enabled CO:RE tracee)`' >> $(release_notes);
+	echo '- `docker pull docker.io/$(PUSH_DOCKER_REPO):$(TARGET_BTFHUB_IMAGE_NAME)-$(RELEASE_TAG)`' >> $(release_notes);
+	echo '- `docker pull docker.io/$(PUSH_DOCKER_REPO):$(TARGET_CORE_IMAGE_NAME)-$(RELEASE_TAG)`' >> $(release_notes);
+	echo '- `docker pull docker.io/$(PUSH_DOCKER_REPO):$(TARGET_NOCORE_IMAGE_NAME)-$(RELEASE_TAG)`' >> $(release_notes);
+
+.PHONY: clean
+clean:
+#
+	$(CMD_RM) .check*
```

Comment:
Hum, I need to add this to other makefiles (to clean .check files and not to rely in Makefile.one only).